deployments/deployment.yaml

```diff
@@ -35,6 +35,24 @@ objects:
             value: "${JWT_MODULE}"
           - name: KEYCLOAK_SERVER
             value: "${KEYCLOAK_SCHEME}://${KEYCLOAK_HOST}:${KEYCLOAK_PORT}${KEYCLOAK_PATH}"
+          - name: COGNITO_APP_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: ocm-app-client
+                key: client_id
+          - name: COGNITO_APP_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: ocm-app-client
+                key: client_secret
+          - name: OAUTH_TOKEN_URL
+            value: ${OAUTH_TOKEN_URL}
+          - name: AMS_URL
+            value: ${AMS_URL}
+          - name: COGNITO_SCOPE
+            value: ${COGNITO_SCOPE}
+          - name: OCM_MODULE
```

Comment:
Should this be `USERS_MODULE` so it's named relevant to the endpoint? Otherwise it might be a bit confusing since OCM is not the endpoint, but rather one implementation of the `/v1/users/` backing service?